client/client-multi/client-c/src/kaa/kaa_notification_manager.c

```diff
@@ -123,6 +124,21 @@ static bool find_notifications_by_topic(void *data, void *context)
     return node->topic_id == *((uint64_t *) context);
 }
 
+static bool sort_topic_by_id(void *node_1, void *node_2)
+{
+    KAA_RETURN_IF_NIL2(node_1, node_2, false);
+    kaa_topic_t *wrapper_1 = (kaa_topic_t *)node_1;
+    kaa_topic_t *wrapper_2 = (kaa_topic_t *)node_2;
```

Comment:
Not an error, but rather recommendation - avoid explicit casting to\from void * pointers since implicit conversion is allowed by C standard and it is type-safe as well. C99 6.3.2.3 Pointers